src/celeritas/io/EventData.hh

```diff
@@ -0,0 +1,78 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/io/EventData.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <array>
+#include <map>
+#include <vector>
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Example of a calorimeter hit.
+ */
+struct EventHitData
+{
+    int volume{0};  //!< Logical volume ID
+    int copy_num{0};  //!< Physical volume copy number
```

Comment:
Is the copy number something we need for validation? I'm guessing not...

Reply:
Right. That's reasonable to keep then. I think for our regular one we use the "flat" one which doesn't have copies.